### ISSUE TITLE
SMS randomly not sent

### ISSUE THREAD

**Issue by arkadiuszkoziel** [signalapp/Signal-Android#12218]
Hello

I have sometimes randomly issues with sending regular sms messages. When I click send, it is trying and after 10-15 seconds I get an error message that sms cannot be sent. When I click on message there is popup message do You want to send it again. When I click send again, then it is sending it without any problems.

Here is error report after such situation.
Last sent sms had this sending problem.

Error report:
https://debuglogs.org/android/5.35.3/4f0cc7c17eb00b14f4c6d31b74f1998970631ca56ba4d9f23edbe8cfac4c7671

Best regards
Arkadius
.

**Comment by clauz9** [comment]
Have you updated to a newer version ? and is this still happening ?

**Comment by arkadiuszkoziel** [comment]
Hello   Yes, I am using latest version of Signal.  In previous versions it was hanging forewer. I need to stop application, start it again and then it sending of SMS was unblocked.  But now it is waiting only 10-15 seconds and giving error. Anyway this is better than before.   Best Regards   Arkadius     
            
          
     
      
       
        Dnia 5 maja 2022 12:53 Claus ***@***.***&gt; napisał(a):
       
    
       
         
 Have you updated to a newer version ? and is this still happening ?

**Comment by erik55** [comment]
This seems to be a duplicate of #9030  and #12382

**Comment by greyson-signal** [comment]
Hi there, given our future plans to [remove SMS support](https://signal.org/blog/sms-removal-android/), I'm gonna close this. Thanks!

**Comment by clauz9** [comment]
boy this was quick 🤣
